mpi/mpi_tests.c

```diff
@@ -0,0 +1,23 @@
+#include <unistd.h>
+#include <stdio.h>
+
+#include "mpi.h"
+
+int HelloWorld() {
```

Comment:
I wonder if we should adapt `t/mpi/hello.c` from flux-core here instead of writing a custom test.

If you feel strongly about using this version, then I'd prefer we use flux-core coding style here, and I have some other comments on this specific code. (braces for functions go on the next line, declaration of `processor` should be at the top of the function, `HelloWorld` should be lowercase separated with underscore (`hello_world`), and also the function always returns 0 so might as well make it return `void` or move it into main since it is so short)

Reply:
Pulled `t/mpi/*.c` into this repo.